src/seismometer/seismogram.py

```diff
@@ -368,9 +372,8 @@ def create_cohorts(self) -> None:
                 )
                 continue
             if not sufficient.all():
-                logger.debug(
-                    f"Some cohorts of {disp_attr} were below censor limit: {', '.join(sufficient[~sufficient].index)}"
-                )
+                log_details = ", ".join(sufficient[~sufficient].index.astype(str))
+                logger.debug(f"Some cohorts of {disp_attr} were below censor limit: {log_details}")
```

Comment:
Should this be a warning rather than a debug, and should we say what happed to that column in the log message? (it looks like we do some category manipulation in line 378

Reply:
will 'elevate' this to a warning in next commit